data/json/items/gun/762x25.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "ppsh",
```

Comment:
Inconsistent field ordering. Every other file under `gun/` follows an identical pattern which I would prefer to maintain. Elsewhere in the JSON this is not so important but it would be unfortunate not to maintain this given 95% of the gun JSON data is migrated to this format